NCrontab/CrontabSchedule.cs

```diff
@@ -100,6 +100,12 @@ public sealed partial class ParseOptions
 
         public static CrontabSchedule? TryParse(string expression) => TryParse(expression, null);
 
+        public static bool TryParse(string expression, out CrontabSchedule? schedule)
+            => (schedule = TryParse(expression, null))!=null;
+
+        public static bool TryParse(string expression, ParseOptions options, out CrontabSchedule? schedule)
+            => (schedule = TryParse(expression, options)) != null;
```

Comment:
Formatting inconsistencies and `options` needs to be nullable to match the type of the argument of [the overload being delegated to](https://github.com/atifaziz/NCrontab/blob/920066b6fba48c2a54e8c31bb2acde1f985d0173/NCrontab/CrontabSchedule.cs#L103):

```suggestion
        public static bool TryParse(string expression, ParseOptions? options, out CrontabSchedule? schedule) =>
            (schedule = TryParse(expression, options)) != null;
```